### ISSUE TITLE
Grid search on finetune

### ISSUE THREAD

**Issue by anamariamocanu** [minimaxir/gpt-2-simple#104]
Does anyone have some thoughts on how to do it? I'm trying to figure out the best learning rate and number of steps for my dataset but not sure how to do it other than running it in a loop.

Thanks!

**Comment by minimaxir** [comment]
The quantitative value of the loss is not super helpful in text quality, therefore grid search would not work well. There are no plans to add it.
